spec.bs

```diff
@@ -601,6 +640,9 @@ A <dfn export>fenced frame config</dfn> is a struct with the following [=struct/
 
   : <dfn>embedder shared storage context</dfn>
   :: null, or an [=fencedframetype/embedder shared storage context=]
+
+  : <dfn>required permissions to load</dfn>
```

Comment:
We don't need this field (yet). This is the point of [=fenced frame config/effective permissions=].

"required permissions to load" as a name doesn't capture the fact that it disables all other permissions anyway.